### PR TITLE
MAISTRA-1316 Prevent SMCP/SMMR/SMM resources from being created in the operator namespace

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

**Review by rcernich — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -69,7 +69,7 @@ compile:
 ################################################################################
 .PHONY: test
 test:
-	go test -v -timeout ${TEST_TIMEOUT} -mod=vendor ./...
```
Comment (dgn) [inline Makefile]: let's make it optional at least. it helps with debugging failures on prow. we can make it dependant on an env var - on prow we already set `CI=prow`

**Review by dgn — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (luksa) [reply Makefile]: oh, I thought we added that mostly by mistake.. I'll remove it from this PR, so we can address it properly

Comment (dgn) [reply Makefile]: Thanks

**Pull request merged**